api/src/models/publication.model.js

```diff
@@ -42,7 +42,23 @@ const publicationSchema = new mongoose.Schema({
     },
     yearPublished: {
         type: Date
-    }
+    },
+    category: {
+        type: {
+            type: String,
+            enum: ['CONFERENCE', 'JOURNAL'],
+            required: true
+        },
+        categoryTitle: {
+            type: String,
+            required: true,
+            minlength: 3
+        },
+        volume: String,
+        issue: String,
+        pages: String,
+        publisher: String
```

Comment:
Also, unclear what these attributes mean. To me it looks like we're referencing a book? If so, we can probably document this internally and describe the schema attributes.

Reply:
I copied these attributes from google scholar when creating publication there. 
The client did not ask for these attributes to be included in publication explicitly, I just think would be a good idea to include here for now. 
May be we can confirm with the client whether he needs this or not, if so then we can doucment this internally, otherwise we could just remove all these attriutes.

Reply:
Got it, @vgoh1 do you think we should keep these and document them somewhere or pull them out altogether?

Reply:
I think keep them here because they're not going to have any other use as far as I can think of